### DISCUSSION TITLE
Welcome to OMG_BADGES Discussions!

### DISCUSSION THREAD

**Discussion by Rich14S** [Rich14S/OMG_BADGES#4] [Q&A] [answered]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rich14S** [comment]
Pra que serve essa sessão?

**Reply by roodriiigooo** [reply]
O objetivo dela é se comunicar com seus colegas, isso pode ser utilizado para diversas tarefas!

**Reply by Rich14S** [reply]
> O objetivo dela é se comunicar com seus colegas, isso pode ser utilizado para diversas tarefas!

Booa!!

**Comment by Rich14S** [comment]
Eu posso criar quantas perguntas quiser?

**Reply by roodriiigooo** [reply]
Sim! você e todas as pessoas podem! Façam com moderação! :D

**Comment by roodriiigooo** [comment, marked as answer]
Acho que posso responder isso como um final de questão.
